packages/keystone/lib/List/index.js

```diff
@@ -1094,6 +1094,24 @@ module.exports = class List {
       originalInput,
       actions: mapKeys(this.hooksActions, hook => hook(context)),
     };
+
+    // Check for isRequired
+    const fieldValidationErrors = this.fields
```

Comment:
This happens _after_ relationships are resolved: https://github.com/keystonejs/keystone-5/blob/16386a2d96e0378e2e2a433c8de3bbce21745515/packages/keystone/lib/List/index.js#L1280

This could result in a related item being created or deleted before the input is validated (and then fails validation)